### PR TITLE
Lesson_2_1 Refactor guessing game

### PR DESCRIPTION
**Pull request by verafes**
Converting stateful class components to function components

### CODE REVIEW

**Comment by AnnaPestova1** [comment]
Hi Vera! 
Could you please handle only 5 guesses, please? Right now I can try and try and try 
image

File: src/GuessControl.jsx [hunk 1]
```diff
@@ -1,47 +1,29 @@
...
-     *  class have the correct `this` object when they run.
-     */
-    this.handleInputChange = this.handleInputChange.bind(this);
-    this.onSubmitGuess = this.onSubmitGuess.bind(this);
-  }
-
-  handleInputChange(event) {
-    this.setState({ currentGuess: event.target.value });
+  const handleInputChange = (event) => {
+    event.preventDefault();
```
Comment (AnnaPestova1) [inline src/GuessControl.jsx]: do you need an `event.preventDefault()` here?

**Review by AnnaPestova1 — APPROVED** [review]
Hi Vera! Great job! Your app is working as the example one. You have one unnecessary preventDefault() and also a stretch goal, if you want to try to handle situation to prevent more guesses than 5. It is not in the scope of the assignment. So, it is if you want to challenge yourself.

File: src/GuessControl.jsx (hunk 1 above)
Comment (verafes) [reply src/GuessControl.jsx]: the line removed

**Review by AnnaPestova1 — APPROVED** [review]
Looks great!

**Pull request merged**